addon/index.js

```diff
@@ -12,7 +12,64 @@
  * ```
  */
 export function proto(target, key, desc) {
-  desc.value = desc.initializer();
+  desc.value = desc.initializer.apply(target);
   desc.initializer = undefined; // unset the initializer
   return desc;
 }
+
+/**
+ * Decorator which merges the value of the class field with the value
+ * of the field on the superclass.
+ *
+ * ```js
+ * class Foo {
+ *   baz = { prop1: 123 };
+ * }
+ *
+ * class Bar {
+ *   @concatenated baz = { prop2: 456 };
```

Comment:
This seem incorrect? I would have expected `@merged` here...

Reply:
ah yep, definitely a typo